project/Build.scala

```diff
@@ -94,11 +96,9 @@ object MacroGLBuild extends MechaRepoBuild {
         mechaNightlyKey.dependsOn(compile in Compile)
     )
   
-  lazy val macrogl: Project = Project(
-    "macrogl",
-    file("."),
-    settings = macroglSettings
-  )
+  lazy val macrogl: CrossProject = CrossProject("macrogl", file("."), CrossType.Full)
+    .settings(macroglSettings)
```

Comment:
This seems to break the build. It is also orthogonal to 2.12 cross compilation, so I would move it to another PR.